modules/drivers/gnss/conf/gnss_conf.pb.txt

```diff
@@ -1,43 +1,43 @@
 data {
-    format: NOVATEL_BINARY
+    format: NEWTONM2_BINARY
```

Comment:
please don't change default gnss_conf.pb.txt. can you help to add a new Newton GPS config? e.g. gnss_conf_newton.pb.txt

Reply:
I have already modified the comment you left, please review again.